src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/ManifestTests.cs

```diff
@@ -125,5 +131,59 @@ public void GivenNonExistentPackRoot_ItIgnoresIt()
 
             pack.Path.Should().Be(defaultPackPath);
         }
+
+        [Fact]
+        public void ItChecksDependencies ()
+        {
+            string MakeManifest(long version, params (string id, long version)[] dependsOn)
```

Comment:
we want to use a string as the version right? Could make the change at the sametime? Looks like a big chunk of the code is to deal with the manifest version

Reply:
@mhutch ?

Reply:
Done